components/brave_wallet/common/eth_request_helper_unittest.cc

```diff
@@ -44,8 +46,11 @@ TEST(EthRequestHelperUnitTest, CommonParseErrors) {
     std::vector<uint8_t> nonce;
     std::vector<uint8_t> ephemeral_public_key;
     std::vector<uint8_t> ciphertext;
-    EXPECT_FALSE(ParseEthDecryptData(error_case, &version, &nonce,
-                                     &ephemeral_public_key, &ciphertext));
+    const auto json = base::JSONReader::Read(error_case);
```

Comment:
No way to call `ParseEthDecryptData` with a invalid json now.
In fact, it wasn't called before this PR (because the code handles `base::unexpected` before)

Reply:
Just drop non-json strings from error_cases

Reply:
Not sure. 
Looks like the other methods in the test should be checked agains an invalid json string.